docs/manual/manual_timeout.rst

```diff
@@ -15,29 +15,20 @@ disarm when a timeout occurs, leaving them in a safe state following a timeout.
 Module Support
 ===============
 
-Speed Modules
-**************
-
-.. include:: all_checked_table.rst
-
-Servo Modules
-**************
-Servo modules do not support the full range of configurable timeout features described in this section. They do support a timeout, and the period of the timeout is configurable.
-The timeout behavior, timeout song playback options, and timeout meaning described in this section are not supported on servo modules. Servo modules will always coast when they timeout.
-
-.. table:: Module Support
-
-	+--------------+------------------------------------+-------------------------------------------------------------------------+
-	| Module       | Feature Supported                  | Notes                                                                   |
-	+--------------+------------------------------------+-------------------------------------------------------------------------+
-	| Vertiq 81-XX | .. centered:: |:warning:|          | A limited subset of timeout features are supported, see the note above. |
-	+--------------+------------------------------------+-------------------------------------------------------------------------+
-	| Vertiq 60-XX | .. centered:: |:warning:|          | A limited subset of timeout features are supported, see the note above. |
-	+--------------+------------------------------------+-------------------------------------------------------------------------+
-	| Vertiq 40-XX | .. centered:: |:warning:|          | A limited subset of timeout features are supported, see the note above. |
-	+--------------+------------------------------------+-------------------------------------------------------------------------+
-	| Vertiq 23-XX | .. centered:: |:warning:|          | A limited subset of timeout features are supported, see the note above. |
-	+--------------+------------------------------------+-------------------------------------------------------------------------+
+To see if your module and firmware style supports this feature, please see our :ref:`supported features table <supported_features_table>`.
+
+.. _triggering_timeout:
+
+Triggering Timeout
+=====================
+Timeouts occur when the module is spinning and has not received a new message or command within its :ref:`timeout period <timeout_period>`. If a module has just rebooted, is 
+coasting, or is braking, the timeout will not expire. The countdown only begins when the module receives a command that makes it start spinning or otherwise applying some drive voltage. 

```

Comment:
should it be "otherwise apply some drive voltage" instead of "applying"? The same way it was "start spinning"